README.md

```diff
@@ -34,7 +34,7 @@ resources:
 Behavior
 --------
 
-### `out`: Manipulate a Github webhook
+### `out.js`: Manipulate a Github webhook
```

Comment:
This should also be just [`out`](https://concourse-ci.org/implementing-resource-types.html#resource-out)